api/Document.json

```
@@ -1599,7 +1599,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": "10"
+              "version_added": true
```

Found that it was actually implemented in IE5, so we can add the version number in!

```suggestion
              "version_added": "5"
```